hurumap/views.py

```diff
@@ -0,0 +1,12 @@
+from django.views.generic import TemplateView
+from django.conf import settings
+
+
+class EmbedView(TemplateView):
```

Comment:
Why do we need this?

Reply:
@kilemensi I want to pass the colorbrewer settings to the iframe so that it can use the color scheme of it's parent page

Reply:
What is wrong with setting the `HURUMAP_THEME` here with the rest of other settings: https://github.com/CodeForAfrica/HURUmap/blob/21e5852e4d0c3b82a615d69141e045a8edcd2191/hurumap/templates/settings_js.html#L7-L10

I think this is what we're doing in Takwimu and then we can access it via `window.HURUMAP_THEME`

Reply:
> What is wrong with setting the `HURUMAP_THEME` here with the rest of other settings:
> 
> https://github.com/CodeForAfrica/HURUmap/blob/21e5852e4d0c3b82a615d69141e045a8edcd2191/hurumap/templates/settings_js.html#L7-L10
> 
> I think this is what we're doing in Takwimu and then we can access it via `window.HURUMAP_THEME`

@kilemensi how do these get into the settings file, if `settings_.html` is included in the iframe but the iframe doesn't have view context with those settings, won't they just be blank?

Reply:
By default, `embed/iframe.html` imports `settings_js.html`: https://github.com/OpenUpSA/wazimap/blob/0aac4583097762ef3ce68fa85ab22b70222ede77/wazimap/templates/embed/iframe.html#L67

Are you changing this behaviour?

Reply:
@kilemensi so I move the `HURUMAP_THEME` to this file, but I don't change the `EmbedView`?

Reply:
I added `HURUMAP_THEME` in `settings_js.html` in Takwimu _without_ `EmbedView` and it worked @phillipahereza hence me asking do we need `EmbedView` _at all_?

Reply:
@kilemensi lemme try but I am not sure how it works